specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,126 @@
+# Service Subcategory
+
+The Service Subcategory is the second-highest-level classification of a [*service*](#glossary:service) based on its core function.  The Service Subcategory (in conjunction with the Service Category) is commonly used for scenarios like analyzing costs across providers and tracking the migration of workloads across fundamentally different architectures.  The Service Subcategory is a child of the Service Category, and each Service Subcategory value must have one and only one Service Category parent. Though a given *service* can have multiple purposes, each *service* should have one and only one Subcategory that best aligns with its primary purpose. 
```

Comment:
I'm open to changing the language for sure... but, following a documentation rule I consider a best practice, I would really want to avoid using the word "sub-categorization" (or any variant) to define a column with the word Subcategory in the name.  How about something like:

> The Service Subcategory is a more granular classification of the Service Category of a _service_, based on its core function.

Reply:
Going with my draft above.

```suggestion
The Service Subcategory is a more granular classification of the Service Category of a [*service*](#glossary:service) based on its core function.  The Service Subcategory (in conjunction with the Service Category) is commonly used for scenarios like analyzing costs across providers and tracking the migration of workloads across fundamentally different architectures.  The Service Subcategory is a child of the Service Category, and each Service Subcategory value must have one and only one Service Category parent. Though a given *service* can have multiple purposes, each *service* should have one and only one Subcategory that best aligns with its primary purpose. 
```